R/tidyContrasts.R

```diff
@@ -11,7 +11,7 @@
 #'
 #' The contrast names will be used as a new column in the tidy output format.
 #'
-#' @param contrastObj A DGEobj or named list of contrast dataframes. (Required)
+#' @param dgeObj A DGEobj or named list of contrast dataframes. (Required)
```

Comment:
@kar-agg-gen I had called this one `contrastObj` because it could be a DGEobj *or* the contrast dataframes. I guess people won't mind if we just change it to `dgeObj` though, the documentation is clear.

Reply:
@klmedeiros-ag I think I should name contrastObj to DGEdata. In all the places I kept DGEdata for parameter where dgeobj or any item from dgeObj is acceptable